hack/util.sh

```diff
@@ -624,26 +423,26 @@ function create_valid_file() {
 function install_router() {
 	echo "[INFO] Installing the router"
 	oadm policy add-scc-to-user privileged -z router --config="${ADMIN_KUBECONFIG}"
-        # Create a TLS certificate for the router
-        if [[ -n "${CREATE_ROUTER_CERT:-}" ]]; then
-            echo "[INFO] Generating router TLS certificate"
-            oadm ca create-server-cert --signer-cert=${MASTER_CONFIG_DIR}/ca.crt \
-                 --signer-key=${MASTER_CONFIG_DIR}/ca.key \
-                 --signer-serial=${MASTER_CONFIG_DIR}/ca.serial.txt \
-                 --hostnames="*.${API_HOST}.xip.io" \
-                 --cert=${MASTER_CONFIG_DIR}/router.crt --key=${MASTER_CONFIG_DIR}/router.key
-            cat ${MASTER_CONFIG_DIR}/router.crt ${MASTER_CONFIG_DIR}/router.key \
-                ${MASTER_CONFIG_DIR}/ca.crt > ${MASTER_CONFIG_DIR}/router.pem
-            ROUTER_DEFAULT_CERT="--default-cert=${MASTER_CONFIG_DIR}/router.pem"
-        fi
-        openshift admin router --config="${ADMIN_KUBECONFIG}" --images="${USE_IMAGES}" --service-account=router ${ROUTER_DEFAULT_CERT-}
-
-        # Set the SYN eater to make router reloads more robust
-        if [[ -n "${DROP_SYN_DURING_RESTART:-}" ]]; then
-            # Rewrite the DC for the router to add the environment variable into the pod definition
-            echo "[INFO] Changing the router DC to drop SYN packets during a reload"
-            oc set env dc/router -c router DROP_SYN_DURING_RESTART=true
-        fi
+	# Create a TLS certificate for the router
+	if [[ -n "${CREATE_ROUTER_CERT:-}" ]]; then
+		echo "[INFO] Generating router TLS certificate"
+		oadm ca create-server-cert --signer-cert=${MASTER_CONFIG_DIR}/ca.crt \
+			--signer-key=${MASTER_CONFIG_DIR}/ca.key \
+			--signer-serial=${MASTER_CONFIG_DIR}/ca.serial.txt \
+			--hostnames="*.${API_HOST}.xip.io" \
+			--cert=${MASTER_CONFIG_DIR}/router.crt --key=${MASTER_CONFIG_DIR}/router.key
+		cat ${MASTER_CONFIG_DIR}/router.crt ${MASTER_CONFIG_DIR}/router.key \
+			${MASTER_CONFIG_DIR}/ca.crt > ${MASTER_CONFIG_DIR}/router.pem
+		ROUTER_DEFAULT_CERT="--default-cert=${MASTER_CONFIG_DIR}/router.pem"
```

Comment:
The alignment looked fine, I was just wondering. Is it possible to break the formatting into a separate commit so it's easier to view the "real" diff?